functions/go/Makefile

```diff
@@ -1,34 +1,116 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+SHELL=/bin/bash
 GOPATH := $(shell go env GOPATH)
+TAG := latest
```

Comment:
To be safe, default for tags should be `dev` not `latest`. The former makes it explicit that this is a developer build not an official release. It's a defense-in-depth in case authz is not set up properly and a developer can push local builds.

Reply:
If we use dev as default tag, then we don't need target `build-dev`. Will update it.